code/game/objects/items/stacks/stack.dm

```diff
@@ -339,7 +339,7 @@
 	return ..()
 
 /obj/item/stack/restock()
-	if(istype(src,/obj/item/stack/cable_coil) || istype(src,/obj/item/stack/medical))
+	if(istype(src,/obj/item/stack/cable_coil) || istype(src,/obj/item/stack/medical) || istype(src,/obj/item/stack/package_wrap))
```

Comment:
I'll implement that but it doesn't fix the associated mommi problem which is why I didn't mention it. Having said that, it's an issue for a different PR altogether.